sdg/components.py

```diff
@@ -45,6 +45,7 @@ def sdg_op(
     model = getenv("model")
     endpoint = getenv("endpoint")
     client = openai.OpenAI(base_url=endpoint, api_key=api_key)
+    pipeline_type = getenv("pipeline_type", "simple")
```

Comment:
This seems like it won't work with the standalone CLI (but I assume I'm missing something here). This only allows us to set the `pipeline_type` as an env variable. I assume this will be set correctly when using the `sdg_serving_data_details` secret with the additional 'pipeline_type' variable. But I don't see how the click.options would ever get used here. Does  `ctx.obj["sdg_pipeline_type"]` somehow handle setting it as an env?

Reply:
Yes this value is being fed through env vars through the sdg serving secret, just like the endpoint/model/etc. 
https://github.com/redhat-et/ilab-on-ocp/blob/95eebca3bbdb162c3a6ce292f41666aabc3c71d6/standalone/standalone.tpl#L1109

See [here](https://github.com/redhat-et/ilab-on-ocp/pull/133/files#diff-7791f5c215c18f7e52873a5c8da37c76ecf97db053b8122bafc77d570d5ca585R630) for the readme docs